### ISSUE TITLE
Postscript previewer showing a white square in windows 10

### ISSUE THREAD

**Issue by santanuchakrabarti** [Alhadis/Atom-PostScript#3]
Thanks for helping to remove the error with Ghostscript executable. Now the error is gone but the postscript previewer is not showing any content, only a white square and there is no error message also.

It seems the package is not at all stable/working in windows platform. :-(

**Comment by Alhadis** [comment]
When you checked out this repository, did you remember to install dependencies via `npm install --production .`…?

Check that `~/.atom/packages/postscript` contains:

~~~
node_modules/atom-live-view
node_modules/mapped-disposable
node_modules/pan-and-zoom
~~~

**Comment by Alhadis** [comment]
@santanuchakrabarti Try updating your checkout again.

**Comment by santanuchakrabarti** [comment]
Hi I have the node modules available in my package and I have updated postscript to the latest version, but still I am getting just a white square in preview on Windows 10 machine.

[Edit]
I am extremely sorry!!! After a bit long wait time I am able to read the .ps file in the preview window. But the problem is I can't scroll down or go to the next page.

**Comment by Alhadis** [comment]
> But the problem is I can't scroll down or go to the next page.

Yup, and that's precisely where I left the previewer because I was getting frustrated by Atom's APIs. 😁 I neglected to mention that this part of the package was never finished (hence why I never published it to Atom's [package registry](https://atom.io/packages)). I saw the amount of work involved and realised I'd be committing myself to a non-trivial project that I would need to develop and maintain instead of other projects I'd prefer to be working on instead… 😕

**Comment by santanuchakrabarti** [comment]
> > But the problem is I can't scroll down or go to the next page.
> 
> Yup, and that's precisely where I left the previewer because I was getting frustrated by Atom's APIs. 😁 I neglected to mention that this part of the package was never finished (hence why I never published it to Atom's [package registry](https://atom.io/packages)). I saw the amount of work involved and realised I'd be committing myself to a non-trivial project that I would need to develop and maintain instead of other projects I'd prefer to be working on instead… 😕

I see. Anyway at least it gives an idea of the content of .ps file without leaving the editor. :-) Thanks for it!!

**Comment by Alhadis** [comment]
My pleasure! Happy that my half-arsed efforts have at least managed to help someone. 😀
